<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="_1_1label_2">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/pic/images/bittersweet.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>441</width>
       <height>171</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="id">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
     <widget class="QLabel" name="password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_box">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_box">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="remember_pw">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="aotu_login">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
